web/src/resolvers/edit-project.resolver.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Resolve, ActivatedRouteSnapshot, Router } from '@angular/router';
+
+import { AuthenticationService } from '../services/authentication.service';
+import { ProjectModel } from '../models/index.model';
+import { ProjectService } from '../services/project.service';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class EditProjectResolver implements Resolve<boolean> {
+
+    constructor(private authService: AuthenticationService,
+                private projectService: ProjectService,
+                private router: Router) {
```

Comment:
I think the formatting of this similar to above would be easier to read...

```typescript
constructor(
   private a: b,
   private c: d
) {
```

Reply:
Sure !!